maven/example-maven-project/pom.xml

```
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example.app</groupId>
+  <artifactId>example-artifact</artifactId>
+  <version>1.1</version>
+
+  <name>example-artifact-name</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
```

Let's add a licenses field to this pom file so we can use it for testing license scanning too

```suggestion

  <licenses>
    <license>
      <name>Apache License, Version 2.0</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  
```

---

addressed in: https://github.com/fossas/example-projects/pull/2/commits/beb6f08330d8d93efd3832da3915eea77fbaf372